aws_doc_sdk_examples_tools/metadata.py

```diff
@@ -406,26 +393,6 @@ def parse(
     return examples, errors
 
 
-def validate_no_duplicate_api_examples(
```

Comment:
This was public for UpdateMirrors, could you put it back?

Reply:
I have no idea why this is missing.